python/samples/getting_started/06-memory-and-embeddings.ipynb

```diff
@@ -532,6 +532,10 @@
    "metadata": {},
    "outputs": [],
    "source": [
+    "from dotenv import load_dotenv\n",
```

Comment:
Thanks for working on this. I'd love to stick with our current paradigm to load AI service/memory connector settings. Similar to how the settings (API keys, endpoints, etc) are handled for AI services like AzureChatCompletion, the underlying memory store also creates `AzureAISearchSettings` and will attempt to load the env vars/.env file vars, and also allow for these values to be overridden via the constructor. We're striving to move away from using `load_dotenv` where possible, so if we introduce this, I don't want to cause confusion.

If one follows the proper set up to use the .env file, the `AZURE_AI_SEARCH_` config values will be used: https://github.com/microsoft/semantic-kernel/blob/main/python/samples/getting_started/CONFIGURING_THE_KERNEL.md

Reply:
hi @moonbox3, I've changed this to now reference the `AzureAISearchSettings` class in the connectors folder to maintain the same paradigm. Thanks for the feedback and let me know if there's anything I should change!

Reply:
@moonbox3, great! Makes sense--so I can completely remove the cell creating the Azure AI Search settings then?

Reply:
That's right. If the cells above, that create the AI service connectors, are passing then that does mean the .env file/env vars are configured and settings are created successfully. 

![image](https://github.com/user-attachments/assets/5cd24c69-f875-4093-a043-46368dce2f32)

Reply:
Great! @moonbox3 tested and changed. Thanks!